src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/ProxyAcceptStream.java

```diff
@@ -170,8 +170,12 @@ private void handlePreferWaitIfNoneMatchRequest(
         short authScope,
         ListFW<HttpHeaderFW> requestHeaders)
     {
-        final String etag = streamFactory.supplyEtag.get();
-
+        String etag = null;
+        HttpHeaderFW etagHeader = requestHeaders.matchFirst(h -> "etag".equals(h.name().asString()));
```

Comment:
Why are we looking for `etag` on the request headers?
Should that be `if-none-match`?